unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Zephir.
+ *
+ * (c) Phalcon Team <team@zephir-lang.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zephir\Test\Logger\Formatter;
+
+use PHPUnit\Framework\TestCase;
+use Zephir\Config;
+use Zephir\Logger\Formatter\CompilerFormatter;
+
+class CompilerFormatterTest extends TestCase
+{
+    /** @var Config $config */
+    private $config;
+
+    public function setUp()
+    {
+        $this->config = new Config();
+    }
+
+    /** @test */
+    public function shouldBeQuietWhenConfigSilent()
+    {
+        $this->config->set('silent', true);
+        $compilerFormatter = new CompilerFormatter($this->config);
+
+        $this->assertSame('', $compilerFormatter->format([]), 'Should return empty string when Config have: silent=true');
+    }
+
+    public function getWarningContext(): array
+    {
+        return [
+            'message' => 'Variable "param1" declared but not used in test\3__closure::__invoke',
+            'context' => [
+                'unused-variable-external',
+                [
+                    'type' => 'parameter',
+                    'name' => 'param1',
+                    'const' => 0,
+                    'data-type' => 'variable',
+                    'mandatory' => 0,
+                    'reference' => 0,
+                    'file' => realpath(__DIR__.'/../../../../../test/closures.zep'),
```

Comment:
Can you please use here something a bit more human friendly

Reply:
Ok :) I'll try to use fixture with as friendly as possible path :)